.eslintrc

```diff
@@ -135,9 +134,7 @@
     "space-before-blocks": [2, "always"],
     "space-in-parens": [2, "never"],
     "space-return-throw-case": 2,
-    "spaced-comment": [2, "never"],
```

Comment:
You might, but this project does not. 

Sent from my iPhone

> On Mar 18, 2016, at 12:05 PM, Jared Lewis <notifications@github.com> wrote:
> 
> In .eslintrc:
> 
> > @@ -135,9 +134,7 @@
> >      "space-before-blocks": [2, "always"],
> >      "space-in-parens": [2, "never"],
> >      "space-return-throw-case": 2,
> > -    "spaced-comment": [2, "never"],
> We always add spaces after comments so it seemed weird to do it differently
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>